lib/deployinator/templates/log_table.mustache

```diff
@@ -3,7 +3,7 @@
         <tr>
             <th>
                 {{# prev_page }}
-                <form action="/log">
```

Comment:
using the [`log_table`](https://github.com/friedenberg/deployinator/blob/bc678d8ff3110b13806fd5c5698fd2f93c516a82/lib/deployinator/helpers.rb) template is currently limited to pages that match the `/log` route. This change allows other pages to use this template and still get the benefit of functioning prev / next buttons.